### PR TITLE
exprtk: add version 0.0.3

### PR DESCRIPTION
**Pull request by toge**
### Summary
Changes to recipe:  **exprtk/0.0.3**

#### Motivation
There are several improvements in 0.0.3.
There is the official license file since 0.0.3.

#### Details
https://github.com/ArashPartow/exprtk/compare/0.0.2...0.0.3

---
- [x] Read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md)
- [x] Checked that this PR is not a duplicate: [list of PRs by recipe](https://github.com/conan-io/conan-center-index/discussions/24240)
- [x] Tested locally with at least one configuration using a recent version of Conan

### CODE REVIEW

**Comment by jcar87** [comment]
The error during validation is:
```
 "Error parsing YAML: while parsing a block mapping\n  in \"<unicode string>\", line 1, column 1:\n    versions:\n    ^\nexpected <block end>, but found '<scalar>'\n  in \"<unicode string>\", line 3, column 14:\n        folder: \"all\"\n                 ^"
§§
```

which is caused by a missing double quote in the yaml file (also reported by the linter, but slightly differently)

File: recipes/exprtk/config.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 versions:
+  "0.0.3:
```
Comment (jcar87) [inline recipes/exprtk/config.yml:2]:
```suggestion
  "0.0.3":
```

**Review by AbrilRBS — APPROVED** [review]
Thanks!

**Review by danimtb — APPROVED** [review]

**Comment by ArashPartow** [comment]
@toge  Thanks for the update, however can you please explain why you felt the need to change the example?

**Comment by ArashPartow** [comment]
@jcar87 wrt: "Feel free to analyse the code, inquire the PR author or voice any other concerns in the PR where this happened: https://github.com/conan-io/conan-center-index/pull/25832"

I already did ( https://github.com/conan-io/conan-center-index/pull/25832#issuecomment-2455437938 ) yet have not heard back from the PR creator.

How should we proceed?  the short of it would be to remove this particular tag from Conan.

**Comment by valgur** [comment]
@ArashPartow Please consider using an `-rc` or `-alpha` suffix when tagging non-final releases to minimize any chance of confusion im the future. It's preferable if the tags can be considered immutable.

**Comment by ArashPartow** [comment]
@valgur that's a good idea, but how about an even better one: We don't add every single random tag from each project into Conan, regardless of suffixes or prefixes. and we instead do what projects like ossfuzz and Conda etc do and make any such PRs pending until users (not just conan admins etc) that have been formally registered to review and ok the PR and not gamify the creation of PRs in such important infra like Conan.  Just an idea.

**Comment by valgur** [comment]
> @valgur that's a good idea, but how about an even better one: We don't add every single random tag from each project into Conan, regardless of suffixes or prefixes. and we instead do what projects like ossfuzz and Conda etc do and make any such PRs pending until users (not just conan admins etc) that have been formally registered to review and ok the PR and not gamify the creation of PRs in such important infra like Conan.  Just an idea.  
> 
> 
> 
> 

Ok, I'm a random user of VTK that wanted to use the latest, presumably the most bug-free, version of ExprTk. Not hypothetically. How am I supposed to know that your latest release is a stable one or not? SemVer is not that complicated and exists as a common convention for a good reason.

**Comment by ArashPartow** [comment]
@valgur  wrt VTK/Paraview I think @spyridon97 is handling that side of things.  https://gitlab.kitware.com/paraview/paraview/-/issues/22554

**Pull request merged**